core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -68,6 +68,7 @@ object ZkUtils {
   val ConfigUsersPath = s"$ConfigPath/users"
   val ProducerIdBlockPath = "/latest_producer_id_block"
 
+
```

Comment:
unneeded new line

Reply:
ZKUtils changes are not required. reverted the chages